### ISSUE TITLE
Card that shows library of all cameras

### ISSUE THREAD

**Issue by bill340** [twrecked/lovelace-hass-aarlo#156]
Is there any possibility to configure a card, that shows only the library of all cameras? 
If not, could you consider that as an enhancement?

**Comment by twrecked** [comment]
To clarify; are you talking about having the multi-camera card default to showing the library view?

**Comment by bill340** [comment]
Yes, that would be a solution. Just to always show the library instead of the last camera picture.
